### DISCUSSION TITLE
inquiry:  Removal of the "gw" Keybind

### DISCUSSION THREAD

**Discussion by Armadillidiid** [LazyVim/LazyVim#1438] [General]
I noticed that the "gw" keybind  has been removed. I can't find it in the docs either.Could you please confirm if this was intentional? If so, could you share the reason behind it?

I found this keybind very helpful and would like to continue using it. While I know Telescope has a similar feature, I prefer the original "gw" implementation.

**Comment by dpetka2001** [comment]
I'm guessing it got removed because it is an internal vim keybinding for formatting lines and there were some concerns raised in other issues. See `:h gw`. You can still use `*` which does the same. Or make a keybinding of your choice to `*N`, which was the behavior of `gw` before it got removed.

**Comment by Armadillidiid** [comment]
Thank you very much. I've created the mapping using your suggestion.

```lua
map('n', 'gw', '*N', { noremap = true, silent = true })
```
